src/App.tsx

```diff
@@ -1,7 +1,37 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export class App extends Component {
+  state = {
+    pressedKey: 'Nothing was pressed yet',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.eventHandler);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.eventHandler);
+  }
+
+  eventHandler = (event: KeyboardEvent) => {
```

Comment:
![image](https://user-images.githubusercontent.com/96409563/223498020-f59ab978-6c1b-46b4-abc7-015503e115c7.png)
```suggestion
  handleClick = (event: KeyboardEvent) => {
```